docker/dockerfiles/artifacts/artifact.Dockerfile

```diff
@@ -43,6 +43,7 @@ RUN --mount=type=cache,target=/root/.m2/ STARROCKS_VERSION=${RELEASE_VERSION} BU
 
 FROM busybox:latest
 ARG RELEASE_VERSION
+ARG TARGETARCH
```

Comment:
`uname -m` might be more appropriate. docker buildx doesn't provide such parameter for multi-arch docker image build.